src/test/resources/liquibase/harness/data/expectedSql/cloudspanner/loadUpdateData.sql

```diff
@@ -0,0 +1 @@
+INVALID TEST -- Liquibase generates SQL without the Zulu timezone
```

Comment:
For a follow-up pull request: We should preferably make this test work, probably by overriding the way that the Spanner Liquibase provider serializes timestamps.